templates/web/base/report/update/moderation_photo_diff.html

```diff
@@ -1,4 +1,4 @@
-[% IF added %]
+[% IF added.size %]
```

Comment:
I think the fix in this commit is great and could be in it's own pull request. That would make it easier to understand and keep it separate from the refactoring in this PR.

Reply:
I can see how in a number of scenarios that would be useful, but I decided on balance it wasn’t worth the extra bureaucracy in this case.

Reply:
Fair enough